models/Callback.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2019-present, [Your company - Change me] All rights reserved.
+ *
+ * You are hereby granted a non-exclusive, worldwide, royalty-free license to use,
+ * copy, modify, and distribute this software in source code or binary form for use
+ * in connection with the web services and APIs provided by  [Your company - Change me].
+ *
+ * As with any software that integrates with the [Your company - Change me] platform, your use of
+ * this software is subject to the [Your company - Change me] Principles and Policies
+ * [your url for policy]. This copyright notice shall be
+ * included in all copies or substantial portions of the software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type { Address } from './Location';
+
+type Result = $ReadOnlyArray<Address>;
+
+export type Callback = (err: string, res: Result) => void;
```

Comment:
This callback type works for searchForLocations callback; we would need a second Callback type for isPresent, where `res` is a boolean instead.